### ISSUE TITLE
VAD version

### ISSUE THREAD

**Issue by sorgfresser** [m-bain/whisperX#201]
If I setup pyannote separately and import your model afterwards, I get 

> Model was trained with pyannote.audio 0.0.1, yours is 2.1.1. Bad things might happen unless you revert pyannote.audio to 0.x.

Just because I am curious: what model did you use (simply `pyannote/segmentation`?) and is it finetuned in any way? Otherwise I would replace it with v2 locally.
~~If it's been finetuned: do you know the default duration used when training the model? Since you don't set `duration` explicitly, the `Inference` object of `pyannote` will use this instead. It would be awesome to know this to be able to convert to ONNX without having to use a dynamic axis for the amount of frames.~~

Edit: it's a `16kHz` model with `duration=5.0` according to the `__dict__` object which should be quite accurate.

**Comment by sorgfresser** [comment]
As noted [here](https://github.com/m-bain/whisperX/issues/193#issuecomment-1523215883), we want to update to newer versions of pytorch. This requires a newer version of pyannote. Is there any way I can contribute to this? I'd love to version bump pyannote to `2.1.1` for example. This would most likely require a new `s3` bucket though.

**Comment by m-bain** [comment]
I see, yes the model is exactly the same as pyannote one, I just s3'd it to avoid the requirement of huggingface token.

You can try to get it working with the newest version, personally I was not able to get newest pyannote working with torch 2.0.
A new s3 model artifact is fine, I can host that easily if you send the weights :).

**Comment by sorgfresser** [comment]
Quick note on this: the latest torch 1 version `1.13.1` (and torchaudio `0.13.1`) works out of the box with pyannote `2.1.1`. Important to note: did not test GPU on this, even though I can not imagine why it would only fail on gpu since the torch interface remains the same.

**Comment by sorgfresser** [comment]
So pyannote is compatible to pytorch v2, but only on the develop branch as of now, there is no official release yet. Do you feel comfortable about adding the develop branch as dependency? There is a lot of stuff going on there and it can not be assured that the branch isn't failing from time to time. Otherwise I would test the most recent pyannote commit and ensure we take this specific commit hash when installing via pip.
